test_unstructured_inference/inference/test_processing_elements.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from unstructured_inference.constants import Source
+from unstructured_inference.inference.elements import Rectangle
+from unstructured_inference.inference.layout import DocumentLayout, LayoutElement, PageLayout
+
+# A set of elements with pdfminer elements inside tables
+deletable_elements_inside_table = [
+    LayoutElement(
+        bbox=Rectangle(0, 0, 100, 100),
+        text="Table with inner elements",
+        type="Table",
+    ),
```

Comment:
is this element left intentionally without a source? Do think that can be a valid test but want to make sure this is the intention.

Reply:
Yeah, was intentional, this is because if something is a table then should be kept